app/serializers/discussion_reader_serializer.rb

```diff
@@ -9,5 +10,13 @@ class DiscussionReaderSerializer < ActiveModel::Serializer
              :read_items_count,
              :read_salient_items_count,
              :last_read_sequence_id,
-             :volume
+             :volume,
+             :is_read
+  def is_read
```

Comment:
Yikes! this will cause an extra db query for every discussion. Can't we check unreadness on the client side?